charts/policies/allow-privilege-escalation-psp/questions.yaml

```diff
@@ -0,0 +1,63 @@
+# This is a Rancher questions file
+---
+
+# General policy questions:
+questions:
+- variable: "metadata.name"
+  type: string
+  default: allow-privilege-escalation-psp
+  required: true
+  description: Name of policy resource
+  group: Settings
+  label: Name of policy resource
+- variable: "kind"
+  type: enum
+  options:
+    - "ClusterAdmissionPolicy"
+    - "AdmissionPolicy"
```

Comment:
In the KubeWarden world, are both these resources namespaced? Normal patterns would indicate `ClusterAdmissionPolicy` is a global resource, not a namespaced resource, so I'm wondering if you can take in any arbitrary `kind` here since providing a global resource as the `kind` would break your templating right?

Might be better to replace this with something that asks a user if they want their policy to be `clusterScoped` instead.

Reply:
Indeed, thanks. Fixed in https://github.com/kubewarden/helm-charts/pull/131/commits/09cb38ef3603fdef7adf0b22c39b0a5ca1a70459.